constants.js

```diff
@@ -69,6 +69,10 @@ const SupportedStandards = {
  * @see https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-2-ERC725YJSONSchema.md
  */
 const ERC725YKeys = {
+	LSP0: {
+		// bytes10(keccak256('ExtensionHandler'))
+		ExtensionHandlerPrefix: '0x21c768241ee86a8d564b',
+	},
 	LSP1: {
 		// bytes10(keccak256('LSP1UniversalReceiverDelegate'))
 		LSP1UniversalReceiverDelegatePrefix: '0x0cfc51aec37c55a4d0b1',
```

Comment:
Same here, can we add the 2 x zero bytes `0x0000` separator.

```suggestion
		LSP1UniversalReceiverDelegatePrefix: '0x0cfc51aec37c55a4d0b10000',
```